app/routes/titiler/umd_glad_dist_alerts.py

```diff
@@ -52,23 +52,24 @@ async def glad_dist_alerts_raster_tile(
         description="Only show alerts until given date.",
     ),
     render_type: RenderType = Query(
-        RenderType.encoded, description="Render true color or encoded tiles"
+        RenderType.encoded,
+        description="Render true color or encoded tiles. Encoded tiles have the alert date and confidence packed in the image RGB channels for front-end interactive use, such as date filtering with supported technologies. Decoding instructions: Alert Date is calculated as `red * 255 + green`, representing days since 2020-12-31. Confidence is calculated as `floor(blue / 100)`, with values of `2` (high) or `1` (low). Intensity is calculated as `mod(blue, 100)`, with a maximum value of `55`.",
     ),
     alert_confidence: Optional[AlertConfidence] = Query(
         AlertConfidence.low,
         description="Show alerts that are at least of this confidence level",
     ),
-    tree_cover_density: Optional[int] = Query(
+    tree_cover_density_threshold: Optional[int] = Query(
         None,
         ge=0,
         le=100,
         description="Alerts in pixels with tree cover density (in percent) below this threshold won't be displayed. `umd_tree_cover_density_2010` is used for this masking.",
```

Comment:
Suggestion, it would be more consistent with line 60 and more readable if phrased as:

"Show alerts in pixels with tree cover density (in percent) greater than or equal to this threshold.

Same for line 68 below. Line 75 is tougher to rephrase. But you can leave these for a later change if you want.

Reply:
Updated, thanks @danscales.